cfn.yml

```diff
@@ -0,0 +1,54 @@
+AWSTemplateFormatVersion: "2010-09-09"
```

Comment:
We already have CFN for this repo: https://github.com/aws/aws-dynamodb-encryption-python/blob/master/cfn/CB.yml

Should we just add it there? (and/or refactor since it's called `CB.yml` but I don't care about that)

Reply:
I'll keep them separate for now since the CB.yml has the infra for the release projects and this poorly named file is only to create a role